shakenfist/images.py

```diff
@@ -141,9 +143,15 @@ def _blob_get(self, url):
                                               'User-Agent': util.get_user_agent()})
 
                 with open(blob_path + '.partial', 'wb') as f:
+                    last_refresh = 0
+
                     for chunk in r.iter_content(chunk_size=8192):
                         f.write(chunk)
 
+                        if time.time() - last_refresh > 5:
```

Comment:
Magic number alert.

Reply:
This is just a lock refresh, the timeout is copied from elsewhere. I'll fix it.